frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.tsx

```diff
@@ -10,12 +10,13 @@ import { RevisionItems } from '../../utils/traffic-splitting-utils';
 
 interface TrafficSplittingModalProps {
   revisionItems: RevisionItems;
+  cancel: () => void;
 }
```

Comment:
we could extend the interface with the underlying `ModalComponentProps` which has these modal related props.
```suggestion
interface TrafficSplittingModalProps & ModalComponentProps {
  revisionItems: RevisionItems;
}
```